test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -186,6 +186,38 @@ describe('Parrable ID System', function() {
         removeParrableCookie();
       });
     });
+
+    describe.only('GDPR consent', () => {
+      let callbackSpy = sinon.spy();
+
+      const config = {
+        params: {
+          partner: 'partner'
+        }
+      };
+
+      const gdprConsentTestCases = [
+        { consentData: { gdprApplies: true, consentString: 'expectedConsentString' }, expected: { gdpr: 1, gdpr_consent: 'expectedConsentString' } },
+        { consentData: { gdprApplies: false, consentString: 'expectedConsentString' }, expected: { gdpr: 0 } },
+        { consentData: { gdprApplies: true, consentString: undefined }, expected: { gdpr: 1, gdpr_consent: '' } },
+        { consentData: { gdprApplies: 'yes', consentString: 'expectedConsentString' }, expected: { gdpr: 0 } },
+        { consentData: undefined, expected: { gdpr: 0 } }
+      ];
+
+      gdprConsentTestCases.forEach((testCase, index) => {
```

Comment:
nice use of test case array

Reply:
Credits to criteo lol, but yeah nice idea